addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -9,7 +9,11 @@ class AccountMoveReversal(models.TransientModel):
     _inherit = "account.move.reversal"
 
     l10n_latam_use_documents = fields.Boolean(readonly=True)
-    l10n_latam_document_type_id = fields.Many2one('l10n_latam.document.type', 'Document Type', ondelete='cascade')
+    l10n_latam_document_type_id = fields.Many2one(
+        'l10n_latam.document.type', 'Document Type', ondelete='cascade',
+        domain="['|', (not l10n_latam_use_documents, '=', 1), ('id', 'in', l10n_latam_available_document_type_ids)]"
```

Comment:
Alternatively, just
```suggestion
        domain="[('id', 'in', l10n_latam_available_document_type_ids)]"
```
as I'm not sure it makes sense to be able to select any document type if `not l10n_latam_use_documents`?

Reply:
qdp says is ok